### DISCUSSION TITLE
Refund

### DISCUSSION THREAD

**Discussion by Biantuadi** [community/community#39649] [GitHub Education] [answered]
Hello,
Last week I've created a issue about refunding cause by mistake I took the one year subscription of GitHub copilot when I have GitHub for student.
So I created issue to be refunded the 15th I got an email from github saying that I have been reimbursed and that I can use the free version of GitHub copilot. I'm using copilot for free but so far i haven't seen the money in my account yet...
Just want to know how many days does the transaction take.
Thanks.

**Comment by DSDmark** [comment, marked as answer]
Hi @Biantuadi,

I know it's not great help, but I guess it could help you. You can check your status here :-[**Personal billing**](https://github.com/settings/billing)

Here is some documentation which give you a rough idea:- [**GitHub docs**](https://docs.github.com/en/get-started/learning-about-github/faq-about-changes-to-githubs-plans#i-already-paid-for-the-yearly-pro-or-team-plan-do-i-get-a-refund-for-the-price-difference)

Also, you can read this, it will give you rough idea about **payment and refund** things.(https://gist.github.com/zachharkey/7950247)

**Comment by Hdt57** [comment]
Please refund my money

**Comment by np110999** [comment]
I have not received my refund yet.

**Reply by brad-wilkes** [reply]
Neither have I. I guess Microsoft needs the money more than we do.

**Comment by appout2sol** [comment]
i have purchased Enterprise With Public Accounts with 21$ on 2nd August 2024 but GitHub Management deducted 50$. I asked them about payment they replied that we were checking the authorization of your credit card and payment will return back within 10 days working. but still We did not get our payment. i have cancelled Enterprise with public accounts and waiting for our payment
